include/capstone/capstone.h

```diff
@@ -48,7 +48,7 @@ extern "C" {
 #endif
 
 // Capstone API version
-#define CS_API_MAJOR 5
+#define CS_API_MAJOR 6
```

Comment:
Please send this as a separate PR, to merge first

Reply:
Yes. Extract it after the 0.6 release.

Reply:
@Rot127 I think it's better to be done *before* the release, to clearly denote that the `next` branch is different from the `v5` one. @kabeor @aquynh what do you think?

Reply:
Ah, my bad. I thought this was the Rizin PR. Too many PRs...

Reply:
https://github.com/capstone-engine/capstone/pull/2104

Reply:
I think it's better to change this as Major 5 Minor 2